test/pkg/integration/setup.go

```diff
@@ -91,7 +92,7 @@ func Setup(ctx context.Context) (string, string, env.Environment, bool, string,
 	flagset.StringVar(&username, "username", os.Getenv("USER_NAME"), "username to execute the tests with. Chooses one automatically if left blank.")
 	flagset.BoolVar(&enterprise, "enterprise", false, "whether to test enterprise features. requires enterprise lisence installed.")
 	flagset.BoolVar(&gitlab, "gitlab", false, "whether to test gitlab integration.")
-	flagset.BoolVar(&parallel, "parallel", false, "Run test features in parallel")
+	flagset.BoolVar(&parallel, "parallel-features", false, "Run test features in parallel")
```

Comment:
`-parallel` overlaps with the equally named built-in go test flag, but that one expects an int value while here we expect a boolean, so renaming this flag such that we can actually set it